Library/Formula/uni2ascii.rb

```diff
@@ -0,0 +1,19 @@
+class Uni2ascii < Formula
+  homepage "http://billposer.org/Software/uni2ascii.html"
+  url "http://billposer.org/Software/Downloads/uni2ascii-4.18.tar.gz"
+  sha256 "9e24bb6eb2ced0a2945e2dabed5e20c419229a8bf9281c3127fa5993bfa5930e"
+
+  def install
+    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
+    system "make"
+    system "MKDIRPROG='mkdir -p' make install"
```

Comment:
@mikemcquaid This won't work. Because if it's passed as an array to Homebrew versioned `system`, the first one would be treated as the executable rather than env. It should be
```ruby
ENV["MKDIRPROG"]="mkdir -p"
system "make", "install"
```